src/observation/encodeMatcher/operators.js

```diff
@@ -24,6 +25,15 @@ const noNullComparisons: OperatorFunction => OperatorFunction = operator => (lef
 // Same as `a > b`, but `5 > undefined` is also true
 const weakGt = (left, right) => left > right || (left != null && right == null)
 
+const handleLikeValue = (v, defaultV) => typeof v === 'string' ? v : defaultV
+
+export const like: OperatorFunction = (left, right) => {
+  const leftV = handleLikeValue(left, '')
+  const rightV = handleLikeValue(right, '(?!)')
```

Comment:
i don't think this is correct. I _think_ right cannot be null, and also the fallback `?!` will be replaced to nothing by likeToRegexp